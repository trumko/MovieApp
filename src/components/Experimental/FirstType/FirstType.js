import React from "react";

import './FirstType.scss';

export default function FirstType({ children }) {
  return <p className="mr_first">{children}</p>;
}
