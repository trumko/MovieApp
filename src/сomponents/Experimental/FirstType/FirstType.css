.mr_first {
  background-color: yellow;
}
