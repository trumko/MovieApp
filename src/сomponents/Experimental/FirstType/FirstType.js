import React from "react";

import './FirstType.css';

export default function FirstType({ children }) {
  return <p className="mr_first">{children}</p>;
}
