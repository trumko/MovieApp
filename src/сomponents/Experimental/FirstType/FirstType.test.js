import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import FirstType from './FirstType';

configure({adapter: new Adapter()});

describe('FirstType', () => {
  it('should render correctly', () => {
    const output = shallow(
      <FirstType>First title</FirstType>
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
  it('should render correctly', () => {
    const output = shallow(
      <FirstType>Second title</FirstType>
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
