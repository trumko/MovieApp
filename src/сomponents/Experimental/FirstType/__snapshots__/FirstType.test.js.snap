// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FirstType should render correctly 1`] = `
<p>
  First title
</p>
`;

exports[`FirstType should render correctly 2`] = `
<p>
  Second title
</p>
`;
